BonovoBluetooth/src/com/bonovo/bluetooth/BonovoBluetoothHandfree.java

```diff
@@ -359,49 +387,59 @@ protected void onResume() {
 			}
 		}
 		mDialpadPressCount = 0;
+		

```

Comment:
I actually did a test of the code earlier today and made a couple outgoing calls and it didn't crash for me.